src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -1232,40 +1304,57 @@ private IWebRTCAdaptor getWebRTCAdaptor() {
 	}
 
 	/**
-	 * Returns filtered broadcast list
-	 * @param offset
-	 * @param size
-	 * @param type
-	 * @return list
+	 * Get filtered broadcast list
+	 * @param offset - starting point of the list
+	 * @param size - size of the return list (max:50 )
+	 * @param type - type of the stream
+	 * @return list of the broadcast objects
 	 */
 	@ApiOperation(value = "Returns filtered broadcast list", notes = "",responseContainer = "List",response = Broadcast.class)
 	@GET
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Path("/broadcast/filterList/{offset}/{size}/{type}")
 	@Produces(MediaType.APPLICATION_JSON)
-	public List<Broadcast> filterBroadcastList(@ApiParam(value = "offset", required = true) @PathParam("offset") int offset,
-											   @ApiParam(value = "size", required = true) @PathParam("size") int size,
-											   @ApiParam(value = "type", required = true) @PathParam("type") String type) {
+	public List<Broadcast> filterBroadcastList(@ApiParam(value = "starting point of the list", required = true) @PathParam("offset") int offset,
+			@ApiParam(value = "size", required = true) @PathParam("size of the return list (max:50 )") int size,
+			@ApiParam(value = "type", required = true) @PathParam("type of the stream") String type) {
```

Comment:
ApiParam and PathParam definitions needs to be exchanged. 
```suggestion
			@ApiParam(value = "size of the return list (max:50 )", required = true) @PathParam("size") int size,
			@ApiParam(value = "type of the stream", required = true) @PathParam("type") String type
```